packages/create-sitecore-jss/src/templates/vue/package.json

```diff
@@ -61,9 +61,9 @@
     "register-service-worker": "~1.7.2",
     "serialize-javascript": "^6.0.1",
     "vue": "^3.2.45",
-    "vue-i18n": "^9.2.2",
+    "vue-i18n": "9.1.6",
     "vue-meta": "3.0.0-alpha.10",
-    "vue-router": "^4.1.6"
+    "vue-router": "next"
```

Comment:
Usually, it's not recommended to use `next` version, because it's unstable. I would suggest you investigate a different solution

Reply:
Hmm, yes you're right but this was the recommended solution (stackoverflow) I found online to get rid of the warning generated by the latest stable version. In that case we would have to fallback to the previous minor version.

Reply:
@sc-addypathania Does your latest change fix an issue? I see that you used fixed version instead of `^`

Reply:
@illiakovalenko yes they do. They fix the warnings in the console during the build which is causing integrated mode to fail.
If we add `^` it will install the latest minor and patch version which causes these warnings. Therefore we need to install this specific version to get rid of the warnings.